### PR TITLE
Mwl 14 deploy first working version to heroku

### PR DESCRIPTION
**Pull request by VladimirKabar**
working

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+development:
+  adapter: postgresql
+  encoding: utf8
+  database: project_development
```
Comment (maciej-lorens) [inline config/database.yml]: this mean that I need the same databse name, change it to <%= ENV["WIZZART_DATABABASE %> and set env variable on your pc

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  pool: 5
+  username: vladimir
```
Comment (maciej-lorens) [inline config/database.yml]: this mean that I need the same username, change it to <%= ENV["WIZZART_USERNAME %> and set env variable on your pc

File: Gemfile [hunk 3]
```diff
@@ -17,7 +17,6 @@ gem 'rails_admin'
 gem 'pg'
 gem 'hirb'
 
-gem 'jquery-rails'
```
Comment (mcksnk) [inline Gemfile]: do we really have to remote it?

**Review by maciej-lorens — CHANGES_REQUESTED** [review]

File: db/seeds.rb [hunk 4]
```diff
@@ -19,16 +19,4 @@
...
-    title = "resource title #{y+1}"
-    description = "resource description #{y+1}"
-    Resource.create!(title: title,
-                     image: File.new(File.join(Rails.root, "/app/assets/test/#{x+y+1}.jpg")),
-                     description: description,
-                     project: project)
 
-  end
-  x += 1
-end
```
Comment (maciej-lorens) [inline db/seeds.rb:34]: instead of removing it, maybe lets upload image from http://placehold.it/

**Review by maciej-lorens — CHANGES_REQUESTED** [review]

File: db/seeds.rb [hunk 5]
```diff
@@ -19,4 +19,14 @@
 p7 = Project.create!(title: "project_title_7", category: "wizualizacje", description: lorem)
 p8 = Project.create!(title: "project_title_8", category: "wizualizacje", description: lorem)
 
+[p1, p2, p3, p4,p5,p6,p7,p8].each do |project|
```
Comment (maciej-lorens) [inline db/seeds.rb:22]: for future, you can use here:

> Project.all.each.do |project|

**Review by maciej-lorens — APPROVED** [review]

**Pull request merged**